docker-compose.dev.yml

```diff
@@ -0,0 +1,13 @@
+version: "3.4"
+
+services:
+  cron-manager:
+    build:
+      dockerfile: "docker/Dockerfile.dev"
+      context: .
+    env_file:
+      - .env
```

Comment:
why do we need .env file in here? `.env` is only needed to provide variables for docker and not related to ENV environment inside containers